index.html

```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Your name</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/Em_Reeds_Code" target="_blank">Em_Reeds_Code</a>
+          </p>
+          <p class="about">I am curious by nature and as a Software Engineer I can now google to my hearts content.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://leonnoel.com/100devs/" target="_blank" title="100Devs">
+                  100Devs
+                </a>
+              </li>
+              <li>
+                <a href="https://www.traversymedia.com/" target="_blank" title="Traversy Media">
+                  Traversy Media
+                </a>
+              </li>
+              <li>
+                <a href="https://www.kevinpowell.co/" target="_blank" title="Kevin Powell">
+                  Kevin Powell
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this comment as well. 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->` 
to `<!-- ________ Your_Name Contributor card END ________ -->`